playbooks/roles/ocp-customization/tasks/powervm_rmc.yaml

```diff
@@ -0,0 +1,106 @@
+- name: Install openshift python module
+  pip:
+    executable: pip3
+    name: openshift
+
+- name: Create powervm-rmc project
+  k8s:
+    name: powervm-rmc
+    api_version: project.openshift.io/v1
+    kind: Project
+    state: present
+
+- name: Create powervm-rmc project
+  k8s:
+    name: powervm-rmc
+    api_version: project.openshift.io/v1
+    kind: Project
+    state: present
+
+- name: Create powervm-rmc serviceaccount
+  k8s:
+    state: present
+    definition:
+      api_version: v1
+      kind: ServiceAccount
+      metadata:
+        name: powervm-rmc
+        namespace: powervm-rmc
+
+- name: Add privileged scc to powervm-rmc serviceaccount
+  shell: "oc adm policy add-scc-to-user -z powervm-rmc privileged -n powervm-rmc"
+
+- name: Deploy powervm-rmc DaemonSet
+  k8s:
+    state: present
+    definition:
+      kind: DaemonSet
+      apiVersion: apps/v1
+      metadata:
+        name: powervm-rmc
+        namespace: powervm-rmc
+      spec:
+        selector:
+          matchLabels:
+            app: powervm-rmc
+        template:
+          metadata:
+            creationTimestamp: null
+            labels:
+              app: powervm-rmc
+          spec:
+            nodeSelector:
+              kubernetes.io/arch: ppc64le
```

Comment:
I'm wondering if we need to have the `arch` node selector since we only support single arch clusters in OCP today.

Reply:
We are ready for the future ;) ... it also protects users that would deploy this to a non-power environment. In that case, no RMC container ppc64le based will be deployed.